snowpack/src/commands/dev.ts

```diff
@@ -1012,7 +1015,7 @@ export async function startServer(
       return result ? result[0] : null;
     },
     onFileChange: (callback) => (onFileChangeCallback = callback),
-    getServerRuntime: (options) => getServerRuntime(sp, options),
+    getServerRuntime: (options) => getServerRuntime(sp, config, options),
```

Comment:
This seems like a breaking API change? Can `config` be the last param?

Reply:
It's not! The public `getServerRuntime` API just accepts `options`. This change is just to pass the config around internally.

Reply:
Ah I see that now; thanks. Carry on!